ibc_public/utils_labels.py

```diff
@@ -0,0 +1,83 @@
+import json
+import sys
+import pandas as pd
+
+import os
+
+_package_directory = os.path.dirname(os.path.abspath(__file__))
+
+ALL_CONTRASTS = os.path.join(_package_directory, '..', 'ibc_data',
+                             'all_contrasts.tsv')
+CONTRAST_COL = 'contrast'
+
+
+def get_labels(contrasts, contrast_col=CONTRAST_COL):
+    """
+    Returns the list of labels for each passed contrast name
+
+    Parameters
+    ----------
+
+    contrasts: list of str
+               each element of the list is a contrast name in the document
+
+    contrast_col: str
```

Comment:
Why do we need this ?

Reply:
Because the names of the contrasts are fields in a column, not column names

Reply:
Yes, but since the contrasts.csv file has fixed column names, this is not subject to change.

Reply:
So you think I should just index by 'contrast' instead of having it assigned at the beginning of the script?

Reply:
yes.

Reply:
Done